### PR TITLE
Port SuriService from dor-services gem

### LINKED ISSUE #3636: Import SuriService from dor-services

### PR DESCRIPTION
**Pull request by mjgiarlo**
## Why was this change made? 🤔

Fixes #3636

And, in so doing, tear out a bunch of functionality that is not even supported in the Suri service anymore.

## How was this change tested? 🤨

CI, and successfully integration-tested in QA

### CODE REVIEW

**Review by jcoyne — APPROVED** [review]

File: app/services/suri_service.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SuriService
```
Comment (jcoyne) [inline app/services/suri_service.rb:3]: Rubocop and I would appreciate some class level docs here.

Comment (mjgiarlo) [reply app/services/suri_service.rb:3]: @jcoyne Oh, right, I forgot about our friend Rubocop.

Comment (mjgiarlo) [reply app/services/suri_service.rb:4]: Got it now. Thanks!

**Comment by mjgiarlo** [comment]
@jcoyne Can you have another look? I made a couple small tweaks and thought you should 👁️ again before we merge this.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**